ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+
+> Please provide a description of the bug / issue, and provide the details below:
+
+====================================================================
+
+###### If the issue is visual, please provide screenshots here
+
+====================================================================
+
+###### Steps to reproduce the problem
```

Comment:
Next week I might take my girl out to the movies,... Oi wait hold on I have no chick!